test/request.test.ts

```diff
@@ -87,16 +87,23 @@ describe('Apify.Request', () => {
         };
 
         request.pushErrorMessage(undefined);
+        // @ts-expect-error Argument of type 'boolean' is not assignable to parameter of type 'string | Error'.
```

Comment:
if those calls do not produce runtime error and are technically supported, maybe we should adjust the method signature to allow them?

maybe we should use just `unknown` in there